### ISSUE TITLE
PlantSEED Genome Viewer

### ISSUE THREAD

**Issue by samseaver** [ModelSEED/PlantSEED#3]
This page will have two tabs:

  "Genome Overview": This will show the metadata for the Genome, as well as a table showing a list of features, their assigned functions, and their subsystems. I am preparing minimal genome objects that contains this information only. Without the sequences, this reduces the object size to <10MB. Each feature ID will link to the Feature viewer.

  "Annotation Overview": This will be similar to the PlantSEED Subsystems table, but it will include the features of the current genome that have been annotated with each role. For a few of the roles, there will be a feature missing because it had not been annotated. This is where we will be able to highlight the gaps in the annotation. I will prepare the data for this table, for each genome, in a separate object.

**Comment by nconrad** [comment]
Updates: 

The front-end for "Genome Overview" has been worked out.  I should add available metadata.

No work on the front-end has been done for "Annotation overview".  I think I'm waiting on data for that one.

**Comment by samseaver** [comment]
Yes, I will add a annotation_summary function to ProbModelSEEDImpl.pm, which, when called, will compare the annotation of the genome with the full list of annotation from the PlantSEED project, and thereby report what's missing. This would report false negatives.

For false positives, we can BLAST each annotated feature against the A. thaliana exemplars that were originally curated with the same function, and report the statistics. It should then be very easy to see if annotation was incorrectly assigned to a non-homologous protein. However, this will be on the back-burner until we get the rest of the web-page sorted out.

**Comment by samseaver** [comment]
Oh. As part of the modeling process, the genome viewer needs a "Reconstruct" button.

**Comment by samseaver** [comment]
I'm considering the idea that a user would need their own front-page (just for plant genomes/models) that would show, in each row, a plant genome and its associated model, but that's superficial at this point and on the back-burner.

**Comment by nconrad** [comment]
ok. reconstruct would be similar to microbes I assume. I'll have to add an annotation form.

Your vision makes sense... it's something we'd need to do for microbes as well before switching to this site. I agree with back-burning it for now.

**Comment by samseaver** [comment]
Update:

All ten PlantSEED genomes have been uploaded with their accompanying sims, I double-checked a few features on a few genomes, and it all seems to be in order so far.
